### PR TITLE
docs header & toc changes

### PR DESCRIPTION
**Pull request by darzu**
work in progress.

Changes:
- default collapsed TOC
- header follows page
- left align text
- TOC in doc
- logo and search placement

Fixes: https://github.com/microsoft/pxt/issues/5980 and https://github.com/microsoft/pxt/issues/5978

Part 1 of: https://github.com/microsoft/pxt/issues/5981

### CODE REVIEW

**Comment by darzu** [comment]
Current progress, lots to go:
image

**Comment by darzu** [comment]
Before:
docs_before

After:
docs_after

Notice:
Search box
hamburger behavior
bread crumbs

**Comment by darzu** [comment]
Fixes: https://github.com/microsoft/pxt/issues/5980 and https://github.com/microsoft/pxt/issues/5978

File: docfiles/tocheader.html [hunk 1]
```diff
@@ -1,36 +1,32 @@
...
+    <div>
+        @sidebarToggle@
+        <div class="item">
+            <div>
+                <a href="/" class="header" tabindex="0" title="Go to editor">
+                    @targetlogo@
+                </a>
+            </div>
+            <div>
+                Microsoft MakeCode
```
Comment (pelikhan) [inline docfiles/tocheader.html]: This template is used by all editor. This should probably some kind of @ macro.

**Comment by pelikhan** [comment]
print only shows one page

**Pull request merged**